package.json

```diff
@@ -0,0 +1,56 @@
+{
+	"name": "trader",
+	"version": "0.0.1",
+	"private": true,
+	"scripts": {
+		"build": "./node_modules/.bin/tsc",
+		"start": "node node_modules/react-native/local-cli/cli.js start",
+		"start:android": "node node_modules/react-native/local-cli/cli.js run-android",
+		"start:ios": "node node_modules/react-native/local-cli/cli.js run-ios",
+		"test": "jest --coverage"
+	},
+	"dependencies": {
+		"react": "16.0.0-beta.5",
+		"react-native": "0.49.5"
+	},
+	"devDependencies": {
+		"@types/jest": "^21.1.5",
+		"@types/react": "^16.0.19",
+		"@types/react-native": "^0.49.5",
+		"@types/react-test-renderer": "^16.0.0",
+		"babel-jest": "21.2.0",
+		"babel-preset-react-native": "4.0.0",
+		"jest": "21.2.1",
+		"react-test-renderer": "16.0.0-beta.5",
+		"ts-jest": "^21.1.4",
+		"typescript": "^2.6.1"
+	},
+	"jest": {
+		"preset": "react-native",
+		"moduleFileExtensions": [
+			"ts",
+			"tsx",
+			"js"
+		],
+		"transform": {
+			"^.+\\.(js)$": "<rootDir>/node_modules/babel-jest",
+			"\\.(ts|tsx)$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
+		},
+		"testRegex": "(/__tests__/.*|\\.(test|spec))\\.(ts|tsx|js)$",
+		"testPathIgnorePatterns": [
+			"\\.snap$",
+			"<rootDir>/node_modules/",
+			"<rootDir>/lib/"
+		],
+		"cacheDirectory": ".jest/cache",
+		"mapCoverage": true,
+		"coverageThreshold": {
+			"global": {
+			  "branches": 100,
+			  "functions": 100,
+			  "lines": 100,
+			  "statements": 100
```

Comment:
I don't feel like we need to mandate a certain amount of test coverage, what's your experiences with this field?

Reply:
I think it would help to maintain certain thresholds for test coverage (obviously not 100%) but it would be good later on (especially when it is being maintained by other people). From my experience, when a lot of refactoring needed to be done, it was very scary making changes to a project with little unit tests because you won't know when you are unintentionally breaking something. It'll just be here to help make sure the quality of our software is good :)

I think we can also integrate code coverage with jenkins if that's something of interest!